src/Api/ApiServiceProvider.php

```diff
@@ -64,6 +66,15 @@ public function register()
             );
         });
 
+        $this->app->bind('flarum.api.proxy_middleware', function () {
+            $config = $this->app->get('flarum.config');
```

Comment:
Shouldn't this be $this->app->make, so that we can fully use DI?

Reply:
Idk, but get works fine. What would be the benefit of using make?

Reply:
Looking at https://github.com/illuminate/container/blob/b5cfeb3b9e9c0e12f663b4101865762d14f795ee/Container.php#L639, make allows for additional parameters (although they aren't needed here), but both ultimately call resolve. That being said, let's stick with make for consisteny purposes since that's what's used across the codebase.